pkg/config/con_test.go

```
@@ -0,0 +1,348 @@
+	result := cl.sortFilesByDepth(files)
+
+	if !reflect.DeepEqual(result, expected) {
+		t.Errorf("Expected %v but got %v", expected, result)
+	}
+}
+
+func TestDownloadRemoteConfig(t *testing.T) {
+	// Create a mock HTTP server
+	mockServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+		w.Write([]byte("mock content"))
```

golangci-lint (1.62.2)

Error return value of `w.Write` is not checked

(errcheck)